### PR TITLE
[FIX] set-toolbar-items postMessage

### PR DESCRIPTION
**Pull request by ggazzo**
We had some refactors and this code was deleted...

image
image

### CODE REVIEW

File: packages/rocketchat-ui-flextab/client/flexTabBar.js [hunk 1]
```diff
@@ -201,6 +201,16 @@ Template.RoomsActionTab.onCreated(function() {
...
 Template.RoomsActionTab.helpers({
 	...commonHelpers,
+	postButtons() {
+		const toolbar = Session.get('toolbarButtons');
+		const buttons = Object.keys(toolbar.buttons).map(key => {
+			return {
+				id: key,
+				...toolbar.buttons[key]
+			};
+		});
```
Comment (tassoevan) [inline packages/rocketchat-ui-flextab/client/flexTabBar.js]: Maybe compact mapper function as

```
id => ({ id, ...toolbar.buttons[id] })
```

File: packages/rocketchat-ui/client/components/header/header.js [hunk 2]
```diff
@@ -104,25 +104,38 @@ Template.header.helpers({
 });
 
 Template.header.events({
-	'click .iframe-toolbar button'() {
+	'click .iframe-toolbar .js-iframe-action'(e) {
 		fireGlobalEvent('click-toolbar-button', { id: this.id });
+		$('button', e.currentTarget).blur();
```
Comment (tassoevan) [inline packages/rocketchat-ui/client/components/header/header.js]: `e.currentTarget.querySelector('body').blur()` should match the required browser supported.

File: packages/rocketchat-ui/client/components/header/header.js [hunk 3]
```diff
@@ -104,25 +104,38 @@ Template.header.helpers({
...
-			if (err) {
-				return handleError(err);
+		return Meteor.call(
+			'toggleFavorite',
+			this._id,
+			!$(event.currentTarget).hasClass('favorite-room'),
+			function(err) {
+				if (err) {
+					return handleError(err);
+				}
```
Comment (tassoevan) [inline packages/rocketchat-ui/client/components/header/header.js]: Could you avoid some extra lines using an arrow function?

Comment (ggazzo) [reply packages/rocketchat-ui/client/components/header/header.js]: yes we can!

**Pull request merged**